app/javascript/components/SearchBar/SearchBar.tsx

```diff
@@ -45,13 +45,14 @@ interface SearchBarProps {
     text: string;
   }
   facetsEnabled?: boolean
+  mobileView?: boolean
 }
 
-export const SearchBar = ({ query = '', relatedSites = [], navigationLinks = [], relatedSitesDropdownLabel = '', alert, facetsEnabled }: SearchBarProps) => {
+export const SearchBar = ({ query = '', relatedSites = [], navigationLinks = [], relatedSitesDropdownLabel = '', alert, facetsEnabled, mobileView }: SearchBarProps) => {
   const [isPaneOpen, setIsPaneOpen] = useState(false);
   const [searchQuery, setSearchQuery] = useState(query);
 
-  const [isMobileView, setMobileView] = useState(false);
+  // const [isMobileView, setMobileView] = useState(false);
```

Comment:
Please remove this comment.

Reply:
@stevenbarragan - Comment has been removed.